script.js

```diff
@@ -1,4 +1,210 @@
+    draws += 1;
+    message = `${playerName}, it's a DRAW! The previous winner ${previousWinner} wins!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+    <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+
+  // if player wins
+  } else if ((playerGuess == 'scissors' && computerChoice == 'paper')
+  || (playerGuess == 'paper' && computerChoice == 'stone')
+  || (playerGuess == 'stone' && computerChoice == 'scissors')) {
+    playerWins += 1;
+    previousWinner = playerName;
+    console.log(previousWinner);
+    message = `${playerName}, you WIN!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+      <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+
+  // if computer wins
+  } else {
+    computerWins += 1;
+    previousWinner = 'computer';
+    console.log(previousWinner);
+    message = `${playerName}, you LOSE!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+      <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+  }
+
+  return message;
+};
+// end of game block
+
+// reverse mode game block
+var reverseMode = function (playerGuess) {
+  var computerChoice = '';
+  var computerNumber = randomNumber();
+  var message = '';
+
+  if (computerNumber == 1) {
+    computerChoice = 'scissors';
+  } else if (computerNumber == 2) {
+    computerChoice = 'paper';
+  } else {
+    computerChoice = 'stone';
+  }
+
+  // if both player and computer choose the same
+  if (playerGuess == computerChoice) {
+    draws += 1;
+    message = `${playerName}, it's a DRAW!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+    <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+
+  // if player wins
+  } else if ((playerGuess == 'paper' && computerChoice == 'scissors')
+  || (playerGuess == 'stone' && computerChoice == 'paper')
+  || (playerGuess == 'scissors' && computerChoice == 'stone')) {
+    playerWins += 1;
+    message = `${playerName}, you WIN!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+      <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+
+  // if computer wins
+  } else {
+    computerWins += 1;
+    message = `${playerName},you LOSE!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+      <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
+  }
+
+  return message;
+};
+// end of reverse mode game block
+
+// game block
+var computerMode = function (playerGuess) {
```

Comment:
Same, try to use helper functions to avoid repeating code across game modes. Consider also using conditionals in a single set of game logic to avoid having to create separate functions for each of the modes, e.g. normal, reverse, and computer mode.